### PR TITLE
158 es private app

### PR DESCRIPTION
**Pull request by mannyrivera2010**
Please Review 
@lsemesky 
@J-Fid 
@seenusuvarna

### CODE REVIEW

File: ozpcenter/api/listing/tests/test_elasticsearch_util.py [hunk 1]
```diff
@@ -24,9 +25,158 @@ def setUpTestData(cls):
         Set up test data for the whole TestCase (only run once for the TestCase)
         """
         data_gen.run()
+        # elasticsearch_util.
```
Comment (seenusuvarna) [inline ozpcenter/api/listing/tests/test_elasticsearch_util.py]: Is this needed as a comment?

Comment (mannyrivera2010) [inline ozpcenter/api/listing/tests/test_elasticsearch_util.py]: for the future when we finish test

**Comment by mannyrivera2010** [comment]
Regarding to AMLNG-158

**Comment by skhtet** [comment]
:+1:

**Pull request merged**